### ISSUE TITLE
HTML template elements are not localized

### ISSUE THREAD

**Issue by tdulcet** [TinyWebEx/Localizer#4]
[HTML template elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template) are not localized as expected:
```html
<template>
	<span data-i18n="__MSG_example__">Please translate me!</span>
</template>
```
The issue is that they are not found by `document.querySelectorAll`:
https://github.com/TinyWebEx/Localizer/blob/f6da4ca47caf7d4c93725363f2bbc994f1d655fd/Localizer.js#L283-L286

This affects my Thunderbird Send add-on, which uses an HTML template element in the popup. See https://github.com/tdulcet/Thunderbird-Send/pull/14.

CC: @DenB10

**Comment by rugk** [comment]
Hmm interesting, it seems this is a feature [of the `template` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template). Does it work if you set [`shadowrootmode`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template#shadowrootmode) to `open`?

**Comment by tdulcet** [comment]
> Does it work if you set [`shadowrootmode`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template#shadowrootmode) to `open`?

No, setting it "open" breaks the template element and produces an exception: `TypeError: can't access property "content", template is null`. The solution might be to iterate over any template elements separately and then perform the query on each one.
